TodoWithControllersAuthJWT/README.md

```diff
@@ -0,0 +1,18 @@
+﻿Added JWT for local authentication and authorization.
+
+Now, TodoWithControllersAuthJWT project is available only for users within Program.validUsers dictionary.
+
+All methods from /api/todos (except GenerateToken) will require token validation
+More than that, 
+- GET /api/todos/{id} will require "user" policy, that is defined as "can_view" claim
+- DELETE /api/todos/{id} will require "admin" policy, that is defined as "can_delete" claim
+
+For demo purposes, Claims are requested on GenerateToken endpoint.
+```
+# pwsh
+$base = "http://localhost:8888"
+$payload = "username=user&password=123456&desiredClaims=delete&desiredClaims=view"
```

Comment:
The claims shouldn't be requested here. It should be generated on the server side.